### PR TITLE
:wrench: Improve organization readme

### PR DESCRIPTION
**Pull request by worthant**
- improved readme structure
- added 1st year tutorial link
- improved contacts & general description

### CODE REVIEW

**Review by DimaThenekov — APPROVED** [review]
Not a bad solution to add spoilers to the main page. This will speed up navigation.

**Review by maxbarsukov — APPROVED** [review]
LGTM 👍

**Review by eliteSufferer — COMMENTED** [review]
Maybe it'd be better to put the disclaimer on top of all readme. In addition, put colours in the text more logically if you want to use them to make accents to some words, now they aren't evident enough.

**Review by owl-from-hogvarts — COMMENTED** [review]
Leaved a couple of comments directly for code lines

File: profile/README.md [hunk 1]
```diff
@@ -1,33 +1,56 @@
...
 
 ### Joining Our Team
-- If you're interested in joining our team, please contact us via email  
-or reach out to our owner `@adagio_sostenuto` on Telegram.
+
+- If you're interested in joining our team, please contact us via [email](morethanjustlabs@gmail.com) :mailbox:
 
 ### Disclaimer
+
 ```python
```
[the remark this answers is not in the document]
Comment (worthant) [reply profile/README.md:40]: fixed it, thanks!)

File: profile/README.md [hunk 2]
```diff
@@ -1,33 +1,56 @@
-## IMTJL - It's More Than Just Labs
+# IMTJL - It's More Than Just Labs
```
[the remark this answers is not in the document]
Comment (worthant) [reply profile/README.md:1]: good suggestion, added it)

**Comment by worthant** [comment]
Good job team 🎉

**Pull request merged**